src/site/antora/modules/ROOT/pages/manual/customloglevels.adoc

```diff
@@ -14,7 +14,7 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 ////
-= Custom Log Levels
+= Levels
```

Comment:
:100: 

This would be the ideal page to roughly explain how levels should be used, by introducing the difference between:

* levels that are meant to draw the attention of the system administrator to a potential problem (`FATAL`, `ERROR`, `WARN`),
* levels that are meant to help system administrators and developers in solving an existing problem (`INFO`, `DEBUG`, `TRACE`).

We should warn the user that the exact meaning of these levels (for app developers) depends on the conventions in their own company.

For library developers we might draft on the most commonly used conventions.